turtlebot3_gazebo/package.xml

```diff
@@ -25,6 +25,8 @@
   <depend>gazebo_ros</depend>
   <exec_depend>gazebo</exec_depend>
   <export>
-    <gazebo_ros gazebo_media_path="${prefix}" gazebo_model_path="${prefix}/models:${prefix}/models/turtlebot3_autorace"/>
+    <gazebo_ros gazebo_media_path="${prefix}"/>
+    <gazebo_ros gazebo_model_path="${prefix}/models"/>
+    <gazebo_ros gazebo_model_path="${prefix}/models/turtlebot3_autorace"/>
```

Comment:
Please, remove <gazebo_ros gazebo_model_path="${prefix}/models/turtlebot3_autorace"/> line. Then, I will merge this PR.

Reply:
Done!